src/components/panels/MmuPanel.vue

```diff
@@ -85,7 +86,7 @@
                     <div class="text-center body-1">{{ statusText }}</div>
                     <mmu-filament-status />
                     <div v-if="showClogDetection" class="text-center">
-                        <mmu-clog-meter width="40%" />
+                        <mmu-clog-meter v-if="hasMmuEncoder" width="40%" />
```

Comment:
What has this if condition todo with the context menu for the gates and why do you need this if condition, when the same one is already included one line above in the `showClogDetection` condition.